### PR TITLE
[INLONG-8511][Manager] Add admin permission of public tenant to all users

### LINKED ISSUE #8511: [Bug][Manager] Public tenant permission denied for new users
### What happened

New users have no permission to the public tenant, unless the admin grant permission to him.
image

### What you expected to happen

All new users and the users from lower version should have permission to public tenant automaticlly.
There is no need to admin grant public permission one by one on the dashboard.

### How to reproduce

Sign up a new user, but admin doesn't grant public tenant permission to him.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8511 

### Motivation

Add admin permission of public tenant to all users

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java [hunk 1]
```diff
@@ -137,4 +139,14 @@ public Boolean delete(Integer id) {
         return true;
     }
 
+    @Override
+    public int saveDefault(String username, String operator) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java:143]: Please change the return type to `Integer` in service or controller layer, thanks.

File: inlong-manager/manager-web/sql/changes-1.8.0.sql [hunk 2]
```diff
@@ -78,6 +78,10 @@ ALTER TABLE tenant_user_role
 CREATE INDEX index_tenant
     ON tenant_user_role (tenant, is_deleted);
 
+INSERT INTO tenant_user_role(username, role_code, tenant, creator)
```
[the remark this answers is not in the document]
Comment (vernedeng) [reply inlong-manager/manager-web/sql/changes-1.8.0.sql:81]: This is a bugfix for 1.8.0.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/user/TenantRoleServiceImpl.java]: fixed, thx

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by fuweng11 — APPROVED** [review]

**Pull request merged**